sdk/mgmtcommon/TestFramework/ClientRuntime.Azure.TestFramework/MockContext.cs

```diff
@@ -73,6 +73,18 @@ internal TestEnvironment TestFxEnvironment
             return context;
         }
 
+        /// <summary>
+        /// Return a new MockContext
+        /// </summary>
+        /// <returns></returns>
+        public static MockContext Start(
+            Type type,
+            [System.Runtime.CompilerServices.CallerMemberName]
+            string methodName= "testframework_failed")
```

Comment:
Why add the methodName if we are going to use it?

Reply:
I think you want to pass it along as well.

Reply:
Hope adding method name does not make the file name so long that it gives you MAX path issues.

Reply:
The method name is not new, existing overload already has it.

Reply:
my bad, thanks for correct this. Looks good.